core/power/CMakeLists.txt

```diff
@@ -2,10 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 add_library(power_table
```

Comment:
Should be removed

Reply:
The state is not ready yet, I will replace in the next PR with proper reworking of state (use HAMT, AMT, structure up-to specs-actor, CBOR-ing).